src/completers/git.zsh

```diff
@@ -130,12 +132,12 @@ _fzf_complete_git() {
 
             *)
                 local treeish
-                if treeish=$(_fzf_complete_git_parse_argument 1 "${arguments%% -- *}" "${(F)git_options_argument_required}") || [[ -n $treeish ]]; then
+                if treeish=$(_fzf_complete_parse_argument "$(( ${arguments[(i)$subcommand]} + 1 ))" 1 "${${(q)arguments[1, ${arguments[(i)--]} - 1][@]}}" "${(F)git_options_argument_required}") || [[ -n $treeish ]]; then
```

Comment:
Why can't it be just 3 like this?
```suggestion
                if treeish=$(_fzf_complete_parse_argument 3 1 "${${(q)arguments[1, ${arguments[(i)--]} - 1][@]}}" "${(F)git_options_argument_required}") || [[ -n $treeish ]]; then
```

Reply:
`${${(q)arguments[1, ${arguments[(i)--]} - 1][@]}}` can be included empty strings. When it is `(git '' checkout master)`, passing constant `3` to the first argument of `_fzf_complete_parse_argument` makes it starts to parse from **3**rd argument, which is `checkout`, but we expect it starts to parse from **4**th argument, which is `master`. Therefore, not a constant value but the next position of the subcommand is used.

Reply:
Thanks for your clear elaboration! As we've discussed in Twitter, any whitespace arguments are just treated normally in git as well. Stick to 3 for clarification here.